langchain4j-open-ai/src/main/java/dev/langchain4j/model/openai/OpenAiEmbeddingModelName.java

```diff
@@ -1,5 +1,8 @@
 package dev.langchain4j.model.openai;
 
+import java.util.HashMap;
+import java.util.Map;
+
 public enum OpenAiEmbeddingModelName {
 
     TEXT_EMBEDDING_3_SMALL("text-embedding-3-small"),
```

Comment:
```suggestion
    TEXT_EMBEDDING_3_SMALL("text-embedding-3-small", 1536),
```
WDYT about putting it here instead of map?

Reply:
Good idea. But there is one thing to consider:

In `OpenAiEmbeddingModel`, String is used to represent modelName instead of OpenAiEmbeddingModelName, so if we want to add dimension to an enum, maybe we need a map in `OpenAiEmbeddingModelName` like below:

```java
    private static final Map<String, Integer> KNOWN_DIMENSION = new HashMap<>(OpenAiEmbeddingModelName.values().length);
    
    static {
        for (OpenAiEmbeddingModelName embeddingModelName : OpenAiEmbeddingModelName.values()) {
            KNOWN_DIMENSION.put(embeddingModelName.toString(), embeddingModelName.dimension());
        }
    }
```

Reply:
Sounds good!